components/hab/src/main.rs

```diff
@@ -486,6 +497,61 @@ fn sub_origin_transfer_ownership(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()>
     command::origin::transfer::start(ui, &url, &token, &origin, &account)
 }
 
+fn sub_accept_origin_invitation(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
+    let origin = m.value_of("ORIGIN").expect("required ORIGIN");
+    let invitation_id: i64 = m.value_of("INVITATION_ID")
+                              .expect("required INVITATION_ID")
+                              .parse()
+                              .expect("INVITATION_ID PARSE");
```

Comment:
If you passed "foobar" as an invitation ID, you'd end up with a nasty crash... we should handle this error in the standard way we handle other errors.

The earlier `expect` is fine because clap won't let us even get down here if that's not set.

Reply:
I'm confused by your concern here since clap is validating the command line input via `{valid_numeric::<i64>}`. If the users passes a value that cannot be coerced from String->i64 it fails like so:

```
error: Invalid value for '<INVITATION_ID>': 'foobar' is not a valid number
```

Reply:
Ah, I missed that it was validated elsewhere! That definitely addresses my concern.

I think I just got triggered by the presence of the `expect`. While we've kind of standardized on the `expect("required X")` pattern for the presence of required arguments, I don't think we have anything corresponding for things with validation functions. 

Perhaps something like `expect("INVITATION_ID should be valid at this point")` or something more pithy?